spec/models/provider_user_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe ProviderUser do
+  describe 'validations' do
+    let!(:existing_provider_user) { create(:provider_user) }
```

Comment:
Required for the `validate_uniqueness_of` check - ShouldaMatchers will use the email address from an existing ProviderUser when testing for uniqueness